### PR TITLE
[test] use delete_many to clean collection before deletion

### PR DESCRIPTION
**Pull request by jordanrfrazier**

### CODE REVIEW

File: ragstack-e2e-tests/e2e_tests/langchain/test_astra.py [hunk 1]
```diff
@@ -458,7 +458,15 @@ def environment():
...
 def close_vector_db(vector_store: VectorStore):
-    vector_store.astra_db.delete_collection(vector_store.collection_name)
+    astra_ref = get_astra_ref()
+    token = astra_ref.token
+    api_endpoint = astra_ref.api_endpoint
+
+    raw_client = LibAstraDB(api_endpoint=api_endpoint, token=token)
+    collection = vector_store.collection_name
+    logging.info(f"Closing vstore; deleting collection: {collection}")
+    raw_client.delete_many(filter=[])
```
Comment (hemidactylus) [inline ragstack-e2e-tests/e2e_tests/langchain/test_astra.py:468]: This should be `filter={}` AFAIK

**Review by zzzming — APPROVED** [review]

**Review by zzzming — APPROVED** [review]
@jordanrfrazier maybe we should add them here too. cc @nicoloboschi 

https://github.com/datastax/ragstack-ai/blob/main/ragstack-e2e-tests/e2e_tests/conftest.py#L67

https://github.com/datastax/ragstack-ai/blob/main/ragstack-e2e-tests/e2e_tests/conftest.py#L83

**Comment by jordanrfrazier** [comment]
> @jordanrfrazier maybe we should add them here too. cc @nicoloboschi
> 
> https://github.com/datastax/ragstack-ai/blob/main/ragstack-e2e-tests/e2e_tests/conftest.py#L67
> 
> https://github.com/datastax/ragstack-ai/blob/main/ragstack-e2e-tests/e2e_tests/conftest.py#L83

Perhaps https://github.com/datastax/ragstack-ai/blob/main/ragstack-e2e-tests/e2e_tests/langchain/test_astra.py#L429 as well

**Comment by jordanrfrazier** [comment]
> > @jordanrfrazier maybe we should add them here too. cc @nicoloboschi
> > https://github.com/datastax/ragstack-ai/blob/main/ragstack-e2e-tests/e2e_tests/conftest.py#L67
> > https://github.com/datastax/ragstack-ai/blob/main/ragstack-e2e-tests/e2e_tests/conftest.py#L83
> 
> Perhaps https://github.com/datastax/ragstack-ai/blob/main/ragstack-e2e-tests/e2e_tests/langchain/test_astra.py#L429 as well

(And the llama ones)

File: .github/workflows/clean-astra-env.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: Clean Astra env
 on:
   workflow_dispatch: {}
   schedule:
-    - cron: '*/30 * * * *'
+    - cron: "*/30 * * * *"
```
Comment (jordanrfrazier) [inline .github/workflows/clean-astra-env.yml:6]: will revert

**Review by nicoloboschi — COMMENTED** [review]
@jordanrfrazier could you refactor the implementation into one single method in conftest.py ?

**Comment by nicoloboschi** [comment]
@jordanrfrazier I'm closing this PR since we have no evidence that it works better, and also we went ahead with the other solution

**Pull request closed without merging**